rust/signed_doc/src/doc_types/mod.rs

```diff
@@ -11,169 +11,63 @@ use crate::DocType;
-    let ids = &[CATEGORY_BASE_TYPE];
-    ids.to_vec()
+    Uuid::from_u128(0x48C2_0109_362A_4D32_9BBA_E0A9_CF8B_45BE)
         .try_into()
         .expect("Failed to convert category base types Uuid to DocType")
 });
 
 /// Proposal document type.
 #[allow(clippy::expect_used)]
 pub static PROPOSAL: LazyLock<DocType> = LazyLock::new(|| {
-    let ids = &[PROPOSAL_BASE_TYPE];
-    ids.to_vec()
+    Uuid::from_u128(0x7808_D2BA_D511_40AF_84E8_C0D1_625F_DFDC)
         .try_into()
         .expect("Failed to convert proposal document Uuid to DocType")
 });
 
 /// Proposal comment document type.
 #[allow(clippy::expect_used)]
 pub static PROPOSAL_COMMENT: LazyLock<DocType> = LazyLock::new(|| {
-    let ids = &[COMMENT_BASE_TYPE, PROPOSAL_BASE_TYPE];
-    ids.to_vec()
+    Uuid::from_u128(0xB679_DED3_0E7C_41BA_89F8_DA62_A178_98EA)
         .try_into()
         .expect("Failed to convert proposal comment document Uuid to DocType")
 });
 
 /// Proposal action document type.
 #[allow(clippy::expect_used)]
 pub static PROPOSAL_SUBMISSION_ACTION: LazyLock<DocType> = LazyLock::new(|| {
-    let ids = &[
-        ACTION_BASE_TYPE,
-        PROPOSAL_BASE_TYPE,
-        SUBMISSION_ACTION_BASE_TYPE,
-    ];
-    ids.to_vec()
+    Uuid::from_u128(0x5E60_E623_AD02_4A1B_A1AC_406D_B978_EE48)
         .try_into()
         .expect("Failed to convert proposal action document Uuid to DocType")
 });
 
-/// Proposal Comment Meta Template document type.
-#[allow(clippy::expect_used)]
-pub static PROPOSAL_COMMENT_META_TEMPLATE: LazyLock<DocType> = LazyLock::new(|| {
-    let ids = &[
-        TEMPLATE_BASE_TYPE,
-        TEMPLATE_BASE_TYPE,
-        COMMENT_BASE_TYPE,
-        PROPOSAL_BASE_TYPE,
-    ];
-    ids.to_vec()
-        .try_into()
-        .expect("Failed to convert proposal comment meta template document Uuid to DocType")
-});
-
 /// Proposal Comment Template document type.
 #[allow(clippy::expect_used)]
-pub static PROPOSAL_COMMENT_TEMPLATE: LazyLock<DocType> = LazyLock::new(|| {
-    let ids = &[TEMPLATE_BASE_TYPE, COMMENT_BASE_TYPE, PROPOSAL_BASE_TYPE];
-    ids.to_vec()
+pub static PROPOSAL_COMMENT_FORM_TEMPLATE: LazyLock<DocType> = LazyLock::new(|| {
+    Uuid::from_u128(0x0B84_24D4_EBFD_46E3_9577_1775_A69D_290C)
         .try_into()
         .expect("Failed to convert proposal comment template document Uuid to DocType")
 });
 
 /// Proposal Template document type.
 #[allow(clippy::expect_used)]
-pub static PROPOSAL_TEMPLATE: LazyLock<DocType> = LazyLock::new(|| {
-    let ids = &[TEMPLATE_BASE_TYPE, PROPOSAL_BASE_TYPE];
-    ids.to_vec()
+pub static PROPOSAL_FORM_TEMPLATE: LazyLock<DocType> = LazyLock::new(|| {
+    Uuid::from_u128(0x0CE8_AB38_9258_4FBC_A62E_7FAA_6E58_318F)
         .try_into()
         .expect("Failed to convert proposal template document Uuid to DocType")
 });
-
-/// -------------- Base Types --------------
-/// Action UUID base type.
-pub const ACTION_BASE_TYPE: Uuid = Uuid::from_u128(0x5E60_E623_AD02_4A1B_A1AC_406D_B978_EE48);
```

Comment:
Why don't we define them like this anymore, they are truly constant.  Why need lazylock and static?